server/bin/pbench-restore-and-unpack-tarballs.py

```diff
@@ -21,10 +21,6 @@
 
         * Creating the controller directory if it doesn't already exist
 
-     c. Create symlink to tar ball in BACKED-UP directory
-
-        * Creating the BACKED-UP directory if it doesn't already exist
-
      d. Create symlink to tar ball in TO-RE-UNPACK directory
```

Comment:
Having removed bullet `c.`, shouldn't you relabel bullet `d.` as `c.`?

Reply:
Fixed.